spec/lib/bank_accounts/account_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe BankAccount::Account do
+#   # alternate method: 
+#   # it "creates BankAccount::Account.new." do
+#   #   BankAccount::Account.new(0,0)
+#   #   expect(BankAccount::Account).to respond_to(:new).with(2).arguments
+#   # end 
```

Comment:
I like this note to yourself. This will be a good project to go back to when refreshing your memory on how to test things in future projects. As such a note to yourself is a great idea.

This is one of the key differences between classwork code and production code.